layouts/partials/page/script.html

```diff
@@ -1,3 +1,22 @@
+<script defer src="/presidium.js"></script>
+{{ if $.Site.Params.enterprise_enabled }}
+<script defer src="{{ $.Site.Params.enterprise_script }}"></script>
```

Comment:
I like this a lot. We can start playing with JS bundles and load them as needed. Seeing as this bundles getting way too big.

Reply:
Agree. There are still a huge amount of improvements we can make. This was just the first bit...more coming next Friday.